build/lib/pgBackRestBuild/Config/Build.pm

```diff
@@ -32,6 +32,7 @@ use constant BLDLCL_CONSTANT_OPTION_GROUP                           => '02-const
 use constant BLDLCL_CONSTANT_OPTION_GROUP_TOTAL                     => 'CFG_OPTION_GROUP_TOTAL';
 use constant BLDLCL_CONSTANT_OPTION                                 => '03-constantOption';
 use constant BLDLCL_CONSTANT_OPTION_TOTAL                           => 'CFG_OPTION_TOTAL';
+use constant BLDLCL_CONSTANT_OPTION_VALUE                           => '04-constantOption';
```

Comment:
Might be nice if the value was '04-constantOptionValue' instead of '04-constantOption' - it's just a bit too similar to '03-constantOption' - but not really a big deal.

Reply:
It should have been `04-constantOptionValue` but I forgot to rename it. Fixed in 059f6b4b23a.